DeepSpeech/Dockerfile.train

```diff
@@ -123,6 +131,24 @@ RUN TASKCLUSTER_SCHEME="https://community-tc.services.mozilla.com/api/index/v1/t
 
 WORKDIR $HOMEDIR
 
+RUN git clone https://github.com/$cc_repo.git $CC_DIR
+
+WORKDIR $CC_DIR
+
+RUN git checkout $cc_branch
+
+WORKDIR $CC_DIR
+
+RUN sed -i '43i \ \ \ \ \ \ \ \ # FIXME: THIS IS WRONG DONT USE VALIDATED.TSV AS A SOURCE \
```

Comment:
Please:
 - generate a local patch from your repo (git diff or git format-patch)
 - use `COPY` to push it into the docker
 - `patch -p1 < file.patch`